plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -1,6 +0,0 @@
-use clap::Parser;
-
-fn main() {
```

Comment:
I think I'm missing something here. As per the [code](https://github.com/thin-edge/thin-edge.io/blob/747177bb2f2610fad1d29fc7f05051524ec94602/crates/core/tedge/src/cli/mod.rs#L52), this apt plugin is not defined as a multi-call binary component, and yet it is a symlink in practice. I just don't get how.

Reply:
Someone with Rust experience will have to look at this.

Reply:
But I'm pretty sure the tedge multi-call uses the `plugins/tedge_apt_plugin/src/lib.rs` file, and not the `main.rs` file

Reply:
Ok, I found the link, the tedge multi call binary calls this function, [run_and_exit](https://github.com/reubenmiller/thin-edge.io/blob/7fee7417fc392c9a7c74c8e714f9f771f8cd661c/plugins/tedge_apt_plugin/src/lib.rs#L341)